NetKAN/KHWearableKISProps

```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version" : "v1.0",
```

Comment:
version 1 of the spec is a special case you you just write `1,` rather than `"v1.0",`